doc/whats-new.rst

```diff
@@ -30,7 +30,7 @@ New Features
   By `Deepak Cherian <https://github.com/dcherian>`_.
 - Added the :py:meth:`count` reduction method to both :py:class:`DatasetCoarsen`
   and :py:class:`DataArrayCoarsen` objects. (:pull:`3500`)
-  By `Deepak Cherian <https://github.com/dcherian/>`_
+  By `Deepak Cherian <https://github.com/dcherian>`_
```

Comment:
:) thanks!

Can you fix the github links in the Documentation section below too?

Reply:
which links?

Reply:
![image](https://user-images.githubusercontent.com/2448579/70263128-8e183480-178d-11ea-9394-188ddf8959a7.png)

I guess the quantile links need to be fixed too.

Reply:
oh, yes, they are missing the trailing underscore and the `:py:meth:` prefix. Done.